langchain4j-google-ai-gemini/src/test/java/dev/langchain4j/model/googleai/GoogleAiGeminiChatModelIT.java

```diff
@@ -510,6 +511,41 @@ void should_handle_enum_type() {
             .isEqualTo("{\"sentiment\": \"POSITIVE\"}");
     }
 
+    @Test
+    void should_handle_enum_output_mode() {
+        // given
+        GoogleAiGeminiChatModel gemini = GoogleAiGeminiChatModel.builder()
+            .apiKey(GOOGLE_AI_GEMINI_API_KEY)
+            .modelName("gemini-1.5-flash")
+            .logRequestsAndResponses(true)
+            .responseFormat(ResponseFormat.builder()
+                .type(JSON)
+                .jsonSchema(JsonSchema.builder()
+                    .rootElement(JsonEnumSchema.builder()
+                        .enumValues("POSITIVE", "NEUTRAL", "NEGATIVE")
+                        .build())
+                    .build())
+                .build())
+            .build();
+
+        // when
+        ChatResponse response = gemini.chat(ChatRequest.builder()
+            .messages(
+                SystemMessage.from(
+                    "Your role is to analyze the sentiment of the text you receive."),
+                UserMessage.from(
+                    "This is super exciting news, congratulations!"
+                )
+            )
+            .build());
+
+        System.out.println("response = " + response);
+
+        // then
+        assertThat(response.aiMessage().text().trim())
```

Comment:
```suggestion
        assertThat(response.aiMessage().text())
```
why is trimming required? seems to work without it